### DISCUSSION TITLE
can't make nothing

### DISCUSSION THREAD

**Discussion by ATSAaiPicture** [carson-katri/dream-textures#596] [Q&A]
When I select the model and enter the text and click create, it gives this error and I can't create anything:

An error occurred while generating. Check the issues tab on GitHub to see if this has been reported before:

OSError("Unable to load weights from checkpoint file for 'C:\\Users\\RADMARD/.cache\\huggingface\\diffusers\\models--stabilityai--stable-diffusion-2-depth\\snapshots\\6330e936a1cc2da7fe149bfba83462003f5e71c7\\vae\\diffusion_pytorch_model.bin' at 'C:\\Users\\RADMARD/.cache\\huggingface\\diffusers\\models--stabilityai--stable-diffusion-2-depth\\snapshots\\6330e936a1cc2da7fe149bfba83462003f5e71c7\\vae\\diffusion_pytorch_model.bin'. If you tried to load a PyTorch model from a TF 2.0 checkpoint, please set from_tf=True.")

**Comment by NullSenseStudio** [comment]
Your model is corrupted. You'll have to redownload it.
